ccleste-1.4.0.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="Celeste Classic C source port for 3DS and PC"
+DESCRIPTION="Celeste Classic is the original free version of Celeste, created in 2016. It was written for the Pico-8 virtual console. The game can be found on Pico-8's website and on itch.io. The original code for the game was written in a form of the Lua programming language tailored to the Pico-8, although the version of Celeste Classic found inside the full Celeste game is written in C#."
+
+HOMEPAGE="https://github.com/lemon32767/ccleste"
+COPYRIGHT="lemon32767"
+LICENSE="GNU GPL v2"
```

Comment:
I'm not sure about LICENSE and COPYRIGHT here. I didn't find any license in the upstream repository.

Reply:
I asked to the author but i have no answer, what i can do it this case?

Reply:
https://www.lexaloffle.com/bbs/?pid=86783#p (shows CC-BY-NC-SA-4.0 in the screenshot)